Services/WindowServer/WindowManager.cpp

```diff
@@ -163,6 +163,22 @@ Gfx::IntSize WindowManager::resolution() const
     return Screen::the().size();
 }
 
+void WindowManager::set_acceleration_factor(double factor)
+{
+    Screen::the().set_acceleration_factor(factor);
+    dbgln("Saving acceleration factor {} to config file at {}", factor, m_config->file_name());
+    m_config->write_entry("Mouse", "AccelFactor", String::formatted("{}", factor));
```

Comment:
Let's call this setting `AccelerationFactor` instead of `AccelFactor`.

Reply:
done!